common/setups/hybrid/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .gmm_system import *
+from .util import *
```

Comment:
Is this really on purpose to have all symbols in this namespace?
Also, as you import all symbols from `sisyphus` below, those will also end up here.
(Just asking. I don't like this too much but if you think this is ok, then just leave it.)

Reply:
as long as `__all__` is set correctly in each file this is fine I guess.

Reply:
this has been the default behaviour in combination with the`__all__`.

I am open to discussing this, but I would make this separate from this pull request and maybe do this in `i6_core`.